pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/route/CreateClinicalDummyKitRoute.java

```diff
@@ -161,7 +161,7 @@ public Object handle(Request request, Response response) {
             if (fixedParticipantId) {
                 randomOncHistoryDetailId =
                         bspDummyKitDao.getRandomOncHistoryForParticipant(ddpInstance.getName(), ddpParticipantId);
-                if (randomOncHistoryDetailId != null) {
+                if (randomOncHistoryDetailId == null) {
```

Comment:
This is why GP's tests are not passing, if a `randomOncHistoryDetailId` was null means we were not able to find any onc histories for GP to use for their tests and that's when we should return. Otherwise we should continue with the tests